--- temp/bruh/arch/x86/entry/syscalls/syscall_64.tbl	2022-09-15 14:17:20.000000000 +0530
+++ linux-5.19.9/arch/x86/entry/syscalls/syscall_64.tbl	2022-11-20 00:33:50.240185732 +0530
@@ -417,3 +417,4 @@
 547	x32	pwritev2		compat_sys_pwritev64v2
 # This is the end of the legacy x32 range.  Numbers 548 and above are
 # not special and are not to be used for x32-specific syscalls.
+548	common	twodcopy		sys_twodcopy
